test/e2e/cluster_samples_operator_test.go

```diff
@@ -33,7 +37,41 @@ func TestClusterSamplesOperator(t *testing.T) {
 	if err != nil {
 		t.Errorf("error waiting for samples resource to appear: %v", err)
 	}
+	return sr
+}
+
+func verifyConditionsCompleteSamplesAdded(sr *samplesapi.SamplesResource) error {
+	return wait.PollImmediate(1*time.Second, 10*time.Minute, func() (bool, error) {
+		if err := sdk.Get(sr); err != nil {
+			return false, nil
+		}
+		if sr.Condition(samplesapi.SamplesExist).Status != corev1.ConditionTrue ||
+			sr.Condition(samplesapi.ImageChangesInProgress).Status != corev1.ConditionFalse {
+			return false, nil
+		}
+
+		return true, nil
+	})
+
+}
+
+func verifyConditionsCompleteSamplesRemoved(sr *samplesapi.SamplesResource) error {
+	return wait.PollImmediate(1*time.Second, 10*time.Minute, func() (bool, error) {
+		if err := sdk.Get(sr); err != nil {
+			return false, nil
+		}
+		if sr.Condition(samplesapi.SamplesExist).Status != corev1.ConditionFalse ||
+			sr.Condition(samplesapi.ImageChangesInProgress).Status != corev1.ConditionFalse {
+			return false, nil
+		}
+
+		return true, nil
+	})
+}
 
+func verifyImagesPresent(t *testing.T) {
+	// as the content will vary over time as openshift/library evolves
+	// sample a random set of imagestream related artifacts
```

Comment:
you've got the content the operator is supposed to install in this repo, so you can reference it and verify it all gets installed.

Reply:
yeah your right ... just makes it more complicated :-)

I'll look at the load fixture utilities in the openshift/origin ext tests and see if that helps me figure out how to load the data via relative file lookups

Reply:
yeah forgot for a sec about bindata in openshift/origin ... going to try to avoid that path, and do relative read